ios/Podfile

```diff
@@ -63,6 +63,21 @@ abstract_target 'Status' do
   end
 
   post_install do |installer|
+    # https://github.com/facebook/react-native/blob/main/packages/react-native/scripts/react_native_pods.rb#L197-L202
+      react_native_post_install(
+        installer,
+        config[:reactNativePath],
+        :mac_catalyst_enabled => false,
+        # :ccache_enabled => true
+      )
```

Comment:
This needs a proper comment and a permanent link, also fixed indent.

Reply:
Took it from react-native 0.73 template here -> https://github.com/facebook/react-native/blob/0.73-stable/packages/react-native/template/ios/Podfile#L47-L55

I could try and summarise what those lines mean or just remove the comment?

Reply:
Oh i see what you mean by permanent link, I'll update